pkg/api/types.go

```diff
@@ -1004,26 +1008,58 @@ type SourceStepConfiguration struct {
 	ClonerefsPath string `json:"clonerefs_path"`
 }
 
+// OperatorStepConfiguration describes the locations of operator bundle information,
+// bundle build dockerfiles, and images the operator(s) depends on that must
+// be substituted to run in a CI test cluster
+type OperatorStepConfiguration struct {
```

Comment:
I think I'd prefer a type

```go
type OperatorStepConfiguration struct {
	Bundles []Bundle `json:"bundles,omitempty"`
	Substitutions []PullSpecSubstitution `json:"substitutions,omitempty"`
}

type Bundle struct {
	DockerfilePath string `json:"dockerfile_path,omitempty"`
	Manifests string `json:"manifests,omitempty"`
}
```

Since we have a 1:1 relationship between the dockerifle path and manifests dir